<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NSRInfoMessageBox</class>
 <widget class="QWidget" name="NSRInfoMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgba(0, 0, 0, 220);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="msgTitle">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	font-weight: bold;
	font-size: 30px;
	font-family: Arial;
	background-color: rgba(0, 0, 0, 0);
	border: none;
}
</string>
     </property>
     <property name="text">
      <string>Message title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="msgLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	font-size: 25px;
	font-family: Arial;
	background-color: rgba(0, 0, 0, 0);
	border: none;
}
</string>
     </property>
     <property name="text">
      <string>Message text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font-weight: bold;
	font-size: 18px;
	font-family: Arial;
	color: white;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0,  x2:0.5, y2:1, stop:0 rgba(27, 49, 67, 255), stop:1 rgba(3, 29, 43, 255));
	border: 2px solid rgba(59, 77, 91, 255);
	border-radius: 10px;	
	min-width: 220px;
	min-height: 40px;
}

QPushButton:pressed {
	color: grey;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0,  x2:0.5, y2:1, stop:0 rgba(12, 24, 52, 255), stop:1 rgba(0, 14, 28, 255));
	border: 2px solid rgba(44, 62, 66, 255);
}

</string>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: white;
	font-weight: bold;
	font-size: 18px;
	font-family: Arial;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0,  x2:0.5, y2:1, stop:0 rgba(27, 29, 27, 255), stop:1 rgba(3, 1, 3, 255));
	border: 2px solid rgba(67, 69, 67, 255);
	border-radius:10px;	
	min-width: 220px;
	min-height: 40px;
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
